pkg/flare/archive_test.go

```diff
@@ -32,7 +32,7 @@ func TestCreateArchive(t *testing.T) {
 	mockConfig.Set("confd_path", "./test/confd")
```

Comment:
We haven't added a test case here, it probably makes sense - you should be able to just use dummy data, even an empty `profile *Profile` to somewhat easily test the functionality.

Reply:
Makes sense to me! I will add tests for `zipPerformanceProfile`. Do you think it also makes sense to unit test the method `CreatePerformanceProfile`? That one does a request against the profile endpoints and creates the `profile` struct from it. I would say that one does not make sense to be tested because it would be completely mocked and it would just be a test of our `apiutil`.